app/scoring/process_scores.py

```diff
@@ -87,11 +87,8 @@ def persist_scores(self, user_uuid):
 
         """
         try:
-            user_session = Sessions(session_uuid=self.value_scores["session-id"])
-            db.session.add(user_session)
-
             user_scores = Scores()
-            user_scores.session_uuid = self.value_scores["session-id"]
+            user_scores.quiz_uuid = self.value_scores["quiz_uuid"]
```

Comment:
Just pointing out again that dash has been dropped and differently this time too.

Reply:
Dropping dashes is just making them consistent. Whenever we're talking about the quiz id from the backend perspective it's quiz_uuid and whenever it's coming from the frontend (e.g. as part of request body) then it's quizId. This is in keeping with style differences between JS and Python. And the quizId is the agreed name with frontend. Have postman tested, with edited request parameters to reflect the change. FYI @NickCallaghan